neofs-local/HOWTO.md

```diff
@@ -169,168 +169,168 @@ drwxr-xr-x    1 root     root        4.0K Sep  2 10:04 ..
 ```
-- list all our files in SGID
+
+### Get information about storage group
+Storage groups are stored as objects in the NeoFS. Storage group meta
+information is stored in the object header. You can obtain it with object
+head request.
+
 ```
-# ab0dd79f-5f52-4846-b863-e78b5a7d84e0
+# CID: 3FYucPGPa2NFZz3J4VTsmoeSxexLxpcwV7CJYX5Asrdi
+# ID: ab0dd79f-5f52-4846-b863-e78b5a7d84e0
 # 
-# Get SGID from previous step and fill command
+# Get data from previous step and fill command
 #
-# neofs-cli sg get --sgid <sgid>
+# neofs-cli object head --cid <cid> --oid <id> --full-headers
 
-bash-5.0# neofs-cli sg get --sgid bb010029-5aa1-45e7-92ab-453bf46a348d --cid 2jXnpixkV3VhXws2rNtJJp5kTumct3TaJYf6h9Tb4zer
+bash-5.0# neofs-cli object get --cid 3FYucPGPa2NFZz3J4VTsmoeSxexLxpcwV7CJYX5Asrdi --oid ab0dd79f-5f52-4846-b863-e78b5a7d84e0 --full-headers
 System headers:
-  Object ID   : bb010029-5aa1-45e7-92ab-453bf46a348d
+  Object ID   : ab0dd79f-5f52-4846-b863-e78b5a7d84e0
   Owner ID    : AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y
-  Container ID: 2jXnpixkV3VhXws2rNtJJp5kTumct3TaJYf6h9Tb4zer
+  Container ID: 3FYucPGPa2NFZz3J4VTsmoeSxexLxpcwV7CJYX5Asrdi
   Payload Size: 0
   Version     : 1
   Created at  : epoch #51, 2019-10-07 16:19:09 +0000 UTC
 Other headers:
-  Link:<type:StorageGroup ID:9d401af5-b89e-4d12-867e-987b0ba87519 > 
+  Link:<type:StorageGroup ID: 67d79328-f33a-4896-ba15-7b2def754f94 > 
+  Link:<type:StorageGroup ID: af5416a6-9fc6-4e9f-92d3-7bb7197cd41f > 
   StorageGroup:<ValidationDataSize:101 ValidationHash:2ejFo82mm1AgywJcHpYsTCuazoT84tmkzFPmrtZ3y6hTbx7tcbMgzaYk8sPgUah6654G1zHEkueAW1xC1p24Wiwq > 
-  Verify:<PublicKey:"\002>p/o\230\223p\372Ep^\274\2608Yc\304\334\216\374\254\343\020\233\244j\2006iLCp" KeySignature:"\004\312Rb{&\350j\374\177\265k\036\007~./\306\226\266\351\302\360Thx\271g\342aT\035\373\345\244\245L-Yo\000!\371\273\236A\356\037q\207\225\303)\030\005mb\226\0031\035\360?\200E" > 
-  SGID:74d7e443-cacd-45cd-b6fe-3646e738d83a 
-  HomoHash:111111111111111APfNaWmCGXxjPYJGZwiKpdX3EUk7VxXnyV7eGd74S7ukPHSnf3F7dPU4kqdaB6DuXv 
-  PayloadChecksum:"\343\260\304B\230\374\034\024\232\373\364\310\231o\271$'\256A\344d\233\223L\244\225\231\033xR\270U" 
-  Integrity:<HeadersChecksum:"<\177oq\363\275\335\230}[\013P^+\323e}B\252Rc'\342\223\273h\035\0312\034\357g" ChecksumSignature:"\004I\315\323H\252\004n[N9\374\354\266\253\022\224\340\377\366\206\323\355\243`;\243\250\357\263uR\310\3337\377\277\234\207\022\365\023\3621B\310!@\035D0\252um\201\345\251\202P\202xBf\334\206" >
+  Verify:<PublicKey:"..." KeySignature:"..." > 
+  HomoHash:...
+  PayloadChecksum:"..." 
+  Integrity:<HeadersChecksum:"..." ChecksumSignature:"..." >
 ```
 
 
-## What about configuration?
+## Default configuration
 
-**By default, environment prepared with next settings:**
+Environment prepared with this default config:
 ```
 bash-5.0# env
 NEOFS_CLI_KEY=/user.key
-NEOFS_CLI_ADDRESS storage01:8080
+NEOFS_CLI_ADDRESS=storage10:8080
```

Comment:
hm. why is that now storage10?

Reply:
CLI app is dependent on `storage10` node, look at `neofs-local/docker-compose.cli.yml` :
```yml
    environment:
      - NEOFS_CLI_KEY=/user.key
      - NEOFS_CLI_ADDRESS=storage10:8080
```

I think it will be better to work with this node in all examples. However there is no difference which node to use for queries. What do you think?

Reply:
oh I see now. True, agreed.